vpr/src/base/read_options.cpp

```diff
@@ -1413,6 +1413,10 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .show_in(argparse::ShowIn::HELP_ONLY);
 
     file_grp.add_argument(args.read_vpr_constraints_file, "--read_vpr_constraints")
+        .help("Writes the floorplanning constraints from placement to the specified XML file.")
```

Comment:
Switch the comments for read and write

Reply:
make a little more detailed, what floorplanning constraints (read floorplanning constraints that packing and placement must respect from the specified XML file)

Reply:
writes out new floorplanning constraints based on current placement to the specified xml